sklearn/kernel_ridge.py

```diff
@@ -6,11 +6,11 @@
 
 import numpy as np
 
+from .base import BaseEstimator, RegressorMixin
+from .metrics.pairwise import pairwise_kernels
+from .linear_model.ridge import _solve_cholesky_kernel
 from .utils import check_X_y
-from sklearn.base import BaseEstimator, RegressorMixin
-from sklearn.metrics.pairwise import pairwise_kernels
-from sklearn.linear_model.ridge import _solve_cholesky_kernel
-from sklearn.utils.validation import check_is_fitted
```

Comment:
The way I see it: tests are like user scripts and so use absolute imports. Library is internal code so it uses relative imports. However, absolute imports should work too.